README.md

```diff
@@ -55,7 +55,7 @@ To store objects in [Amazon Simple Storage Service (S3)](https://docs.aws.amazon
 Note that `prefix` must include the leading slash.
 
 [PUT](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html) and [GET](https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html) are supported. 
-If you make a POST request, a UUID will be generated for the object key and will put that object. There is an `ObjectLocation` header in the response.
+If you make a POST request, a UUID will be generated for the object key and will put that object. There is an `Location` header in the response.
```

Comment:
I would rephrase 'and will put that object' -> 'and the object will be created in the same manner as a PUT request'